persist-cli/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxConstants.java

```diff
@@ -113,6 +113,14 @@ private InheritedTypeReferenceConstants() {
 
     public static final String GENERATED_SOURCE_DIRECTORY = "generated";
     public static final String PATH_CONFIGURATION_BAL_FILE = "database_configuration.bal";
+    public static final String VERSION_PROPERTIES_FILE = "version.properties";
```

Comment:
Can we move these configurations to PersistToolsConstants since these are not used for the bal files?